backend/src/main/java/org/sefglobal/core/model/University.java

```diff
@@ -28,4 +28,8 @@
     @NotNull
     @Column(name = "status", length = 10)
     private String status = "ACTIVE";
```

Comment:
Add status values to a constants class and refer from there instead of hardcoding